server/src/test/java/org/elasticsearch/search/aggregations/bucket/histogram/InternalAutoDateHistogramTests.java

```diff
@@ -49,16 +49,16 @@
     public void setUp() throws Exception {
         super.setUp();
         format = randomNumericDocValueFormat();
-        AutoDateHistogramAggregationBuilder aggregationBuilder = new AutoDateHistogramAggregationBuilder("_name");
-        // TODO[PCS]: timezone set automagically here?
-        roundingInfos = aggregationBuilder.buildRoundings();
     }
 
     @Override
     protected InternalAutoDateHistogram createTestInstance(String name,
                                                        List<PipelineAggregator> pipelineAggregators,
                                                        Map<String, Object> metaData,
                                                        InternalAggregations aggregations) {
+        AutoDateHistogramAggregationBuilder aggregationBuilder = new AutoDateHistogramAggregationBuilder("_name");
```

Comment:
I don't think we need to create a builder here anymore. These tests don't currently set the time zone in the builder anyway so `aggregationBuilder.timeZone()` will always be `null` which implicitly means the time zone will be treated as UTC. As mentioned before we should add a test that does set the time zone to test its treated correctly but that doesn't need to be part of this PR so I think we can remove this line and just pass in `null` below?